erpnext/stock/doctype/stock_entry_detail/stock_entry_detail.json

```diff
@@ -472,11 +473,18 @@
    "label": "Reference Purchase Receipt",
    "options": "Purchase Receipt",
    "read_only": 1
+  },
+  {
+   "fieldname": "reference_project",
```

Comment:
Why fieldname is `reference_project`? `project` is used everywhere.

Reply:
done